sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -370,4 +370,22 @@ class QueryExecutionErrorsSuite extends QueryTest
         "If necessary set spark.sql.ansi.enabled to false to bypass this error.")
     }
   }
+
+  test("DIVIDE_BY_ZERO - SPARK-38724: can't divide by zero") {
```

Comment:
Let's be consistent to other tests and not put JIRA id:
```suggestion
  test("DIVIDE_BY_ZERO: can't divide an integer by zero") {
```